templates/identity/email-password/register-response.html.ejs

```diff
@@ -1,63 +1,40 @@
-<!DOCTYPE html>
-<html lang="en">
-<head>
-  <meta charset="utf-8"/>
-  <meta name="viewport" content="width=device-width, initial-scale=1"/>
-  <title>You are signed up</title>
-  <link rel="stylesheet" href="/.well_known/css/styles/main.css" type="text/css">
-</head>
-<body>
-  <header>
-    <a href="/"><img src="/.well_known/css/images/solid.svg" alt="[Solid logo]" /></a>
-    <h1>Community Solid Server</h1>
-  </header>
-  <main>
-    <h1>You are signed up</h1>
-    <p>
-      <strong>Welcome to Solid.</strong>
-      We wish you an exciting experience!
-    </p>
+<h1>You are signed up</h1>
```

Comment:
```suggestion
<h1>You've been signed up</h1>
```

Does that sound better?

Reply:
In my head they are so close together I can't tell the difference 😉 . I'll just copy it.